source/developers-guide/plugin-quick-start/index.md

```diff
@@ -117,195 +141,119 @@ In many cases you might want to modify the template. Shopware makes use of the S
+       <author>shopware AG</author>
+       <compatibility minVersion="5.3.0"/>
+   
+       <changelog version="1.0.0">
+           <changes lang="de">Erstveröffentlichung</changes>
+           <changes lang="en">First release</changes>
+       </changelog>
+   </plugin>
 ```
-    public function getVersion()
-    {
-        return '1.0.0';
-    }
-
-    public function getLabel()
-    {
-        return 'Slogan of the day';
-    }
-
-    public function install()
-    {
-        $this->subscribeEvent(
-            'Enlight_Controller_Action_PostDispatchSecure_Frontend',
-            'onFrontendPostDispatch'
-        );
-
-        $this->createConfig();
-
-        return true;
-    }
-```
-
-The `getVersion` method just needs to return your plugin version as a string. If you later upload your plugin to the store, Shopware will be able to offer customers updates of your plugin depending on the version. 
+These are only "metadata" and optional.
 
 <div class="alert alert-info">
 <strong>Use a valid version</strong>
 
 The returned version has to be compatible with the [php version_compare](http://php.net/version_compare) function, otherwise, Shopware cannot detect any possible updates.
 </div>
 
-`getLabel` should return the name of your plugin in a human readable way - `SwagSloganOfTheDay` might be a bit technical, so we choose `Slogan of the day`.
-
-These are the only "metadata" methods your actually should implement, everything else is optional.
-
-### The install method
+## The plugin base file
+The main entry point of every plugin is the plugin base file in your plugin directory `SwagSloganOfTheDay.php`. This is placed in `custom/plugins/SwagSloganOfTheDay`.
 
-As we want our plugin to actually do something, we should add the `install` method in the next step. This method will be executed once during installation:
+ In the `SwagSloganOfTheDay.php` file, we create a simple class:
 
-```
- public function install()
+```php
+ <?php
+ 
+ namespace SwagSloganOfTheDay;
+ 
+ use Shopware\Components\Plugin;
+ 
+ class SwagSloganOfTheDay extends Plugin
  {
-     $this->subscribeEvent(
-         'Enlight_Controller_Action_PostDispatchSecure_Frontend',
-         'onFrontendPostDispatch'
-     );
-
-     $this->createConfig();
-
-     return true;
+ 
  }
 ```
 
-3 things happen here:
- * we subscribe to an event
- * we call the createConfig method
- * we return a success flag
-
-In order to subscribe to an event, we use the `$this->subscribeEvent` helper method from the bootstrap base class. The first parameter is the event to subscribe to, in this case `Enlight_Controller_Action_PostDispatchSecure_Frontend`. As mentioned above, this event will be triggered every time **after** a controller in the frontend was executed. The second parameter is the callback function that should be called - `onFrontendPostDispatch`
+It's important to extend the `Shopware\Components\Plugin` class, as it holds a lot of helper functions you will need for plugin development.
```

Comment:
actually the base class doesn't not contain "many" helpers :see_no_evil:

Reply:
d